contracts/vault/Stanley.sol

```diff
@@ -540,4 +546,12 @@ abstract contract Stanley is
 
     //solhint-disable no-empty-blocks
     function _authorizeUpgrade(address) internal override onlyOwner {}
+
+    function addGuardian(address _guardian) external onlyOwner {
+        PauseManager.addGuardian(_guardian);
+    }
+
+    function removeGuardian(address _guardian) external onlyOwner {
```

Comment:
Please add this method to IStanleyInternal

Reply:
ok